app/src/androidTest/java/com/github/se/polyfit/data/repository/MealRepositoryTest.kt

```diff
@@ -0,0 +1,85 @@
+import com.github.se.polyfit.data.remote.firebase.MealFirebaseRepository
+import com.github.se.polyfit.data.repository.MealRepository
+import com.github.se.polyfit.model.meal.Meal
+import com.github.se.polyfit.model.meal.MealOccasion
+import com.github.se.polyfit.model.nutritionalInformation.NutritionalInformation
+import com.google.android.gms.tasks.Tasks
+import com.google.firebase.firestore.DocumentReference
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class MealRepositoryTest {
+
+  private lateinit var mealRepository: MealRepository
+  private lateinit var mealFirebaseRepository: MealFirebaseRepository
+
+  @Before
+  fun setup() {
+    mealFirebaseRepository = mockk()
```

Comment:
you got the mock repo working? :D nice

Reply:
It works now. I think the version of mockk we were using was too old and couldn't deal correctly with tasks.